software/authbox/badgereader_wiegand_gpio.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2018 Ace Monster Toys. All Rights Reserved.
```

Comment:
-2019?

Reply:
This is correct as the code first started in 2018, thus making 2018 the beginning of the expressed time of copyright (and correctly set the date for expiration releasing it into the commons).